frontend/src/pages/Dashboard.jsx

```diff
@@ -1,14 +1,24 @@
+import { useEffect, useState } from "react";
 import DashboardCard from "../components/DecisionCard";
 
 function Dashboard() {
+  const [decisions, setDecisions] = useState([]);
+  useEffect(() => {
+    fetch("http://localhost:5000/decisions-cards")
```

Comment:
shouldn't be /users/${someUserId}/decisions ? a dashboard should display the decision attached to the current user